app/src/main/res/values-de/strings.xml

```diff
@@ -320,5 +320,12 @@
     <string name="notif_success_sending">Nachricht gesendet nach %1$s!</string>
     <string name="read_by">Gelesen von</string>
     <string name="message_information_title">Nachricht Information</string>
-    <string name="message_room_changed_privacy">Room type changed to: %1$s by %2$s</string>     <!--TODO - Add proper translation-->
+    <!--TODO - Add proper translation-->
```

Comment:
Try to put the `<!-- TODO Add proper translation-->` in the side of the string so we can follow the same "pattern" that we're doing for all the TODOS in this file (as well as for the other `strings.xml` files too)

P.s: `<!--TODO - Add proper translation-->` instead of `<!--TODO - Add proper translation-->`

Reply:
Thought it was better to put above the line since we could have a longer string and thus that comment would be missed out. That's fine though... Putting them at the side